extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+( function () {
+	/**
+	 * @returns {number} The A/B test variant number for the Toolbar A/B test
+	 */
+	ve.init.wikia.getToolbarABTestVariantNumber = function () {
```

Comment:
Where the experiment is active depends on how the url filter is configured in Optimizely, so there is no need to check for it here:

If the experiment is active _and_ you are in the test group, getToolbarABTestVariantNumber will return 1, if it is inactive or you are not in the test group, it will return 0.